client/my-sites/comments/main.jsx

```diff
@@ -10,21 +10,33 @@ import { localize } from 'i18n-calypso';
 import Main from 'components/main';
 import PageViewTracker from 'lib/analytics/page-view-tracker';
 import DocumentHead from 'components/data/document-head';
+import CommentList from './comment-list';
+import CommentNavigation from './comment-navigation';
 
 export class CommentsManagement extends Component {
 
 	static propTypes = {
 		basePath: PropTypes.string,
+		siteSlug: PropTypes.string,
+		status: PropTypes.string,
 		translate: PropTypes.func,
 	};
 
 	render() {
-		const { translate, basePath } = this.props;
+		const {
+			basePath,
+			siteSlug,
+			status,
+			translate,
+		} = this.props;
 		return (
-			<Main className="comments">
+			<Main className="comments" wideLayout>
 				<PageViewTracker path={ basePath } title="Manage Comments" />
 				<DocumentHead title={ translate( 'Manage Comments' ) } />
-				<div>Hello World!</div>
+				<div className="comments__primary">
+					<CommentNavigation siteSlug={ siteSlug } status={ status } />
+					<CommentList />
```

Comment:
probably good enough to let this not be a separate component, instead to map over something like `this.props.comments`

actually I'd really like to see this be a list of comment ids instead of comments too (for loading and performance reasons) such that it looks like this…

```js
<CommentNavigation { ...{ siteSlug, status } } />
{ this.props.comments.map( ( { siteId, commentId } ) => (
	<CommentDetail { ...{ siteId, commentId } } />
) ) }
```